debian/frr.pam

```diff
@@ -1,4 +1,4 @@
 # Any user may call vtysh but only those belonging to the group frrvty can
 # actually connect to the socket and use the program.
-auth	sufficient	pam_permit.so
-account	sufficient	pam_rootok.so
+auth	sufficient  pam_permit.so
```

Comment:
whitespace got changed here from tab to spaces :D

Reply:
Damn those different editors, will fix this back 😬

Reply:
Fixed.